java/core/src/main/java/co/worklytics/psoxy/impl/RESTApiSanitizerImpl.java

```diff
@@ -331,6 +344,404 @@ private Object sanitize(Endpoint endpoint, Object jsonResponse) {
     }
 
 
+    private static boolean transformApplies(Transform transform, Object document) {
```

Comment:
All of these new code is from `SanitizerUtils`; so not sure if want to move it or it was a bad paste -but code is still there

Reply:
yeah, you're correct; I think combination of merge + my not having fully split out all the tests. Done now.